Feat: import GeoDataFrames

svenruf opened the pull request:
added trackintel/io/from_geopandas.py: Created functions that warp pd.rename to import GeoDataFrames into the trackintel model
added tests for these functions in tests/io/test_io.py and datasets for these tests in tests/data (geojson files)

the commits regarding the test_pypi file can be ignored as they were deleted. Unfortunately I could not get rid of these commits (also the hard reset did not remove them...) 

the issue with the locations (multiple geometry) is related to the issues [867](https://github.com/geopandas/geopandas/issues/867) / [1275](https://github.com/geopandas/geopandas/issues/1275) in Geopandas

hong2223 commented on tests/io/test_io.py:
```diff
@@ -79,3 +123,14 @@ def test_trips_from_to_postgis(self):
+        trps_csv['started_at'] = trps_csv['started_at'].apply(lambda d: d.isoformat().replace(' ', 'T'))
+        trps_csv['finished_at'] = trps_csv['finished_at'].apply(lambda d: d.isoformat().replace(' ', 'T'))
+        trps_csv['started_at'] = trps_csv['started_at'].apply(lambda d: d.replace('+00:00', 'Z'))
+        trps_csv['finished_at'] = trps_csv['finished_at'].apply(lambda d: d.replace('+00:00', 'Z'))
```
same as above, here it is a bit more confusing..

hong2223 commented on trackintel/io/from_geopandas.py:
```diff
@@ -0,0 +1,203 @@
+
+    return lcs
+
+def tours_from_gpd(gdf, user_id='user_id',started_at='started_at', finished_at='finished_at', origin_destination_location_id='origin_destination_location_id', journey='journey', mapper={}):
```
for this function, there are no tests available? Maybe add an issue saying that test for tours are missing in both read_csv and from_geopandas

hong2223 reviewed (commented):
Nice job! Before we can merge, maybe we need to add some more descriptions to the functions (see the specific comments).
We are also trying to organize the file structures in the 'test' folder. The general idea is to imitate the file structure of the corresponding modules and to create a class for each function. You can see the file 'test_positionfixes.py' and issue #23 as examples. 
So here, maybe create a separate test file named `test_from_geopandas.py`.

svenruf commented:
Updated the PR with the points discussed in our meeting:
- comments on the hardcoded datetime modifications
- cleaned up the structure of the tests
Anything to add?

hong2223 commented:
Nice job!
